Samples/AudioPreprocessing/WinUI/AudioPreprocessing/AudioPreprocessing/ViewModel/PreprocessViewModel.cs

```diff
@@ -0,0 +1,149 @@
+﻿using AudioPreprocessing.Model;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Windows.Graphics.Imaging;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+using Windows.Storage.Streams;
+
+namespace AudioPreprocessing.ViewModel
+{
+    public class PreprocessViewModel : INotifyPropertyChanged
+
+    {
+        private string audioPath;
+        private string imagePath;
+        private SoftwareBitmap melSpectrogramImage;
+
+        public PreprocessViewModel()
+        {
+            PreprocessModel preprocessModel = new PreprocessModel();
+            audioPath = preprocessModel.AudioPath;
+            imagePath = preprocessModel.MelSpecImagePath;
+        }
+
+        public async Task<SoftwareBitmap> GetSampleBitmap()
+        {
+            SoftwareBitmap softwareBitmap;
```

Comment:
can be removed as a local, and I believe that the return can be placed directly in the using:

return await decoder.GetSoftwareBitmapAsync();

Reply:
Cleaned